lib/internal/test_runner/runner.js

```diff
@@ -203,10 +199,15 @@ class FileTest extends Test {
     this.reportStarted();
     this.#handleReportItem(ast);
   }
+  reportStarted() {}
   report() {
-    this.reportStarted();
+    if (!this.skipReporting) {
+      super.reportStarted();
```

Comment:
If the no-op `reportStarted()` is dropped, the `super` can be dropped here.

Reply:
see https://github.com/nodejs/node/pull/46440#discussion_r1096573795